### ISSUE TITLE
import css files not working

### ISSUE THREAD

**Issue by zhigang1992** [storybookjs/presets#84]
following guide at 

https://gist.github.com/shilman/bc9cbedb2a7efb5ec6710337cbd20c0c

reproduce repo

https://github.com/zhigang1992/bug-free-fiesta

Regarding CSS not working, I've found that

```js
import './testFails.css';
import styles from './test.module.css';

// This works
export const Text = () => <div className={styles.testClass}/>;
// This does not work
// export const Text = () => <div className="testClass"/>;
```

Only css-modules works, but not importing style directly.

@mrmckeb

**Comment by sunaurus** [comment]
#70 added `include: [configDir]` on [line 72 of processCraConfig.ts](https://github.com/storybookjs/presets/pull/70/files#diff-80b3f28ef184f3c7a4ac83b1974c3dc6R72).

If I remove this line, my css starts working again. It looks like [webpack requires every css file to match at least one condition in the include array](https://webpack.js.org/configuration/module/#ruleinclude). Since the include array only has the .storybook/ directory in it, any css that is in my src/ or node_modules/ or anywhere else is getting ignored. 

If I remove the include line completely, my styles work without problems.

**Comment by hugomassing** [comment]
Same problem here with rc.11!

**Comment by mrmckeb** [comment]
Hi @sunaurus, good pickup. I think the fix is to also add the CRA directory (`src`) in there - does that work for you?

If so, I can make a fix tomorrow and the speedy @shilman can get it out.

**Comment by zhigang1992** [comment]
@mrmckeb We might need to add node_modules as well since some of the CSS does come from third-party dependencies.

**Comment by sunaurus** [comment]
I can confirm that adding both node_modules/ and src/ fixes the styles in my projects. Only src/ is not enough.

**Comment by shilman** [comment]
Published as `@storybook/preset-create-react-app@1.5.1`
